js-pkg/client/package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "tsup --watch",
     "format": "prettier --write \"**/*.{ts,tsx,js,jsx}\""
   },
-  "version": "0.0.9",
```

Comment:
even though technically it's not breaking semver because we are in a `0.` version, maybe we should make this a `0.1.0` release because it makes breaking changes to the interface?

Reply:
I'd be down to move to a 0.1.0